rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -560,6 +569,14 @@ public void internalAddNodeToCore(RMNode configuredNode) {
             return;
         }
 
+        //noinspection ConstantConditions
+        if(isNumberOfNodesLimited() && isMaximumNumberOfNodesReached()) {
```

Comment:
It looks like you don't have applied formatting (no space between `if` and `(`).